modules/vocabularies/src/main/frontend/src/bioportalApiKey.jsx

```diff
@@ -159,9 +159,9 @@ export function BioPortalApiKey(props) {
         </Grid>
       </> }
 
-      <Dialog onClose={() => {setDisplayPopup(false)}} open={displayPopup}>
-         <DialogTitle disableTypography>
-           <Typography variant="h4" className={classes.dialogTitle}>Change BioPortal API key</Typography>
+      <Dialog onClose={() => {setDisplayPopup(false)}} open={displayPopup} maxWidth="sm" fullWidth>
```

Comment:
`sm` still leaves plenty of blank space. I would make it smaller:

```suggestion
      <Dialog onClose={() => {setDisplayPopup(false)}} open={displayPopup} maxWidth="xs" fullWidth>
```